### PR TITLE
add info on downgrading dbt with pip

### PR DESCRIPTION
**Pull request by sophiad96**
## Description & motivation
I had a user reach out asking how to downgrade the version of dbt they were on using the CLI. They had used pip for installation. We already have instructions for changing versions of dbt with homebrew, but we didn't for pip (only for upgrading) so I wanted to add those to the docs as well!

## Pre-release docs
Is this change related to an unreleased version of dbt?
- [ ] Yes: please
    - update the base branch to `next`
    - add Changelog components: `<Changelog>[New/Changed] in v0.x.0</Changelog>`
    - add links to the "New and changed documentation" section of the latest [Migration Guide](../website/docs/docs/guides/migration-guide)
- [X] No: please ensure the base branch is `current`
- [ ] Unsure: we'll let you know!

### CODE REVIEW

**Comment by annafil** [comment]
I'm so curious about why they needed to downgrade 💭 !

**Comment by sophiad96** [comment]
Great question @annafil! This user needed to downgrade because they had packages installed in their project that were not compatible with v0.20.1.

File: website/docs/docs/get-started/pip-install.md [hunk 1]
```diff
@@ -32,22 +34,36 @@ Plugins:
...
+### Upgrade adapters
 
 To upgrade a specific adapter plugin:
 
 ```shell
 pip install --upgrade dbt-<adapter>
 ```
 
-### Install dbt-core only
+### Install dbt Core only
```
Comment (runleonarun) [inline website/docs/docs/get-started/pip-install.md]:
```suggestion
### Install dbt-core only
```

File: website/docs/docs/get-started/pip-install.md [hunk 2]
```diff
@@ -47,7 +49,21 @@ If you're building a tool that integrates with dbt Core, you may want to install
 ```shell
 pip install dbt-core
 ```
+### Upgrade or downgrade dbt Core versions
+
+To upgrade dbt to its latest version, use:
```
Comment (runleonarun) [inline website/docs/docs/get-started/pip-install.md]:
```suggestion
You can upgrade or downgrade versions of dbt Core by using an upgrade command on the Command line (CLI). For more information, see [Best practices for upgrading in Core versions](/docs/dbt-versions/core#best-practices-for-upgrading).

To upgrade dbt to the latest version:
```

File: website/docs/docs/get-started/pip-install.md [hunk 3]
```diff
@@ -47,7 +49,23 @@ If you're building a tool that integrates with dbt Core, you may want to install
 ```shell
 pip install dbt-core
 ```
+### Upgrade or downgrade dbt Core versions
+
+You can upgrade or downgrade versions of dbt Core by using an upgrade command on the Command line (CLI). For more information, see [Best practices for upgrading in Core versions](/docs/dbt-versions/core#best-practices-for-upgrading).
```
Comment (nghi-ly) [inline website/docs/docs/get-started/pip-install.md]:
```suggestion
You can upgrade or downgrade versions of dbt Core by using the `--upgrade` option on the command line (CLI). For more information, see [Best practices for upgrading in Core versions](/docs/dbt-versions/core#best-practices-for-upgrading).
```

File: website/docs/docs/get-started/pip-install.md [hunk 4]
```diff
@@ -47,7 +49,23 @@ If you're building a tool that integrates with dbt Core, you may want to install
 ```shell
 pip install dbt-core
 ```
+### Upgrade or downgrade dbt Core versions
```
Comment (nghi-ly) [inline website/docs/docs/get-started/pip-install.md]:
```suggestion
### Change dbt Core versions
```

Comment (nghi-ly) [reply website/docs/docs/get-started/pip-install.md]: sugg for something shorter. maybe "Update" but that could be synonymous to upgrade for a lot of people.

non blocking.

File: website/docs/docs/get-started/pip-install.md [hunk 5]
```diff
@@ -47,7 +49,23 @@ If you're building a tool that integrates with dbt Core, you may want to install
...
+
+You can upgrade or downgrade versions of dbt Core by using an upgrade command on the Command line (CLI). For more information, see [Best practices for upgrading in Core versions](/docs/dbt-versions/core#best-practices-for-upgrading).
+
+To upgrade dbt to the latest version:
+
+```
+pip install --upgrade dbt-core
+```
+
+To downgrade to an older version, you'll want to use the command above while specifying the version you want to use. As an example:
```
Comment (nghi-ly) [inline website/docs/docs/get-started/pip-install.md]:
```suggestion
To downgrade to an older version, specify the version you want to use. This command can be useful when you're resolving package dependencies. As an example:
```

File: website/docs/docs/get-started/pip-install.md [hunk 6]
```diff
@@ -47,7 +49,23 @@ If you're building a tool that integrates with dbt Core, you may want to install
...
+
+```
+pip install --upgrade dbt-core==0.19.0
+```
 
+The `upgrade` flag is used to change versions of dbt, regardless of whether you're upgrading or downgrading via pip.
```
Comment (nghi-ly) [inline website/docs/docs/get-started/pip-install.md]:
```suggestion

```

Comment (nghi-ly) [reply website/docs/docs/get-started/pip-install.md]: sugg: i don't think we need this sentence anymore if my sugg for the first sentence in the first paragraph is accepted. 

reason: redundant

or, at least, remove the passive voice in the sentence

**Review by nghi-ly — CHANGES_REQUESTED** [review]
thanks for adding this new content, @runleonarun ! sounds like it would be useful! 

left a few suggs. lemme know if you have questions on any of them!

**Review by nghi-ly — APPROVED** [review]

**Pull request merged**